doc/test/run.t

```diff
@@ -8,5 +8,15 @@ is fine, but you then need to update the list of such exceptions below.
 
   $ DUNE_LANG=$(dune internal latest-lang-version)
   $ grep '(lang dune' ../*.rst | grep -v "$DUNE_LANG"
+  ../coq.rst:  (lang dune 3.17)
+  ../coq.rst:  (lang dune 3.17)
+  ../foreign-code.rst:  (lang dune 3.17)
   ../hacking.rst:``(lang dune 2.7)`` in their ``dune`` project file to use it.
+  ../hacking.rst:   (lang dune 3.17)
+  ../instrumentation.rst:   (lang dune 3.17)
+  ../instrumentation.rst:   (lang dune 3.17)
+  ../melange.rst:  (lang dune 3.17)
+  ../sites.rst:   (lang dune 3.17)
+  ../sites.rst:  (lang dune 3.17)
+  ../sites.rst:  (lang dune 3.17)
```

Comment:
These probably need to be updated, as written in the description above.

Reply:
OK, done.